Fabric.Authorization.Persistence.SqlServer/EntityModels/Role.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using Fabric.Authorization.Domain.Models;
+
+namespace Fabric.Authorization.Persistence.SqlServer.EntityModels
+{
+    public class Role : ITrackable, ISoftDelete
+    {
+        public Role()
+        {
+            Groups = new List<Group>();
+            ChildRoles = new List<Role>();
+            /*Permissions = new List<Permission>();
+            DeniedPermissions = new List<Permission>();*/
+        }
+        public int Id { get; set; }
+        public int? ParentRoleId { get; set; }
```

Comment:
The domain model requires a Guid for ParentRole, I guess we'll have to handle that in the model mapping.